ts/components/helpers/withValidatedEmail.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * A HOC to display the WrappedComponent when the email is valid , otherwise the RemindEmailValidationOverlay will be displayed
```

Comment:
```suggestion
 * A HOC to display the WrappedComponent when the email is validated, otherwise the RemindEmailValidationOverlay will be displayed
```